client/package.json

```diff
@@ -4,6 +4,7 @@
   "proxy": "http://localhost:5000",
   "private": true,
   "dependencies": {
+    "@react-navigation/native-stack": "^6.9.9",
```

Comment:
Did you use this? Looking through your code looks like you used `useNavigate` instead. If you didn't use this dependency, please remove it